bundle-size/app.js

```diff
@@ -42,6 +42,20 @@ async function getBuildArtifactsFile(github, filename) {
   });
 }
 
+/**
+ * Get the list of OWNERS that can approve this PR.
+ *
+ * @param {github} github an authorized GitHub API object.
+ * @throws {Error} on any error.
+ * @return {!Array<string>} an array of OWNERS GitHub usernames.
+ */
+async function getOwners(github) {
+  return (await getBuildArtifactsFile(github, 'OWNERS'))
+      .trim()
+      .split('\n')
+      .filter(line => !line.startsWith('#') && !!line);
```

Comment:
Shouldn't `!!line` be checked before `!line.startsWith('#')`?

Reply:
Done